tests/resources/functional/snowflake/functions/stats/test_approx_top_k_1.sql

```diff
@@ -3,4 +3,4 @@
 SELECT approx_top_k(col1) AS approx_top_k_col1 FROM tabl;
 
 -- databricks sql:
-SELECT APPROX_TOP_K(col1) AS approx_top_k_col1 FROM tabl;
+SELECT APPROX_TOP_K(col1, 1) AS approx_top_k_col1 FROM tabl;
```

Comment:
Default value within Databricks is K=5

https://docs.databricks.com/aws/en/sql/language-manual/functions/approx_top_k